workspaces/lightspeed/packages/app/e2e-tests/utils/fileUpload.ts

```diff
@@ -93,3 +93,12 @@ export async function validateFailedUpload(page: Page) {
   await expect(alertHeader).toBeHidden();
   await expect(alertText).toBeHidden();
 }
+
+export async function verifyAlertPopup(
+  state: 'visible' | 'hidden',
+  ...locators: Locator[]
+) {
```

Comment:
This function name is misleading `verifyAlertPopup`, where the name implies you are verifying alert popups but the function works for any set of locators not just alerts.


You could change it to something like below:


```suggestion
export async function assertVisibilityState(
  state: 'visible' | 'hidden',
  ...locators: Locator[]
) {
```